x/auth/keeper/keeper_test.go

```diff
@@ -95,6 +96,9 @@ func (suite *KeeperTestSuite) TestSupply_ValidatePermissions() {
 func (suite *KeeperTestSuite) TestInitGenesis() {
 	suite.SetupTest() // reset
 
+	txFees1, txFees2 := big.NewInt(0), big.NewInt(0)
+	txFeesSum := (big.NewInt(0).Add(txFees1, txFees2)).String()
```

Comment:
Why not just set `txFeesSum` to `big.NewInt(0)` ?

Reply:
Pushed, thanks